src/components/App.js

```diff
@@ -57,6 +58,22 @@ const App = React.createClass({
     );
   },
 
+  getInitialState() {
+    return {
+      vertical: verticalLayoutBreakpoint.matches
+    };
+  },
+
+  componentDidMount() {
+    verticalLayoutBreakpoint.onchange = () => this.setState({
```

Comment:
`verticalLayoutBreakpoint` has both `addEventListener` and `removeEventListener`, but then I'll need to store the callback function somewhere, so I decided to go with `onchange = `